benchmark/opperf/nd_operations/nn_basic_operators.py

```diff
@@ -29,58 +29,132 @@
+                                                        {"data": (2**17, 3, 128, 128),
+                                                         "num_hidden": 64,
+                                                         "weight": (64, 3 * 128 * 128),
+                                                         "bias": (64,),
+                                                         "flatten": False}],
+                                                warmup=warmup,
+                                                runs=runs)
+
+        # Dropout benchmarks
+        dropout_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "Dropout")],
+                                                     run_backward=True,
+                                                     dtype=dtype,
+                                                     ctx=ctx,
+                                                     profiler=profiler,
+                                                     inputs=[{"data": (2**15, 3, 256, 256),
+                                                              "p": 0.5,
+                                                              "mode": "always"},
+                                                             {"data": (2**28, 16),
+                                                              "p": 0.5,
+                                                              "mode": "always"}],
+                                                     warmup=warmup,
+                                                     runs=runs)
+        # BatchNorm benchmarks
+        batchnorm_benchmark_res = run_performance_test([getattr(MX_OP_MODULE, "BatchNorm")],
+                                                       run_backward=True,
+                                                       dtype=dtype,
+                                                       ctx=ctx,
+                                                       profiler=profiler,
+                                                       inputs=[{"data": (2**15, 3, 256, 256),
+                                                                "gamma": (3,),
+                                                                "beta": (3,),
+                                                                "moving_mean": (3,),
+                                                                "moving_var": (3,)},
+                                                               {"data": (2**14, 3, 10000, 10),
+                                                                "gamma": (3,),
+                                                                "beta": (3,),
+                                                                "moving_mean": (3,),
+                                                                "moving_var": (3,)}],
+                                                       warmup=warmup,
+                                                       runs=runs)
+    else:
```

Comment:
It seems the only difference between the if and else branch is the `inputs` argument. Can we only generate different inputs in the if/else branch and pass them to the same operator function?